ci-operator/config/openshift/release/openshift-release-master__ci-4.9.yaml

```diff
@@ -201,13 +225,27 @@ tests:
     env:
       TEST_SUITE: openshift/conformance/parallel
     workflow: openshift-e2e-openstack-ipi
+- as: e2e-openstack-techpreview-parallel
+  cron: 0 2,10,18 * * *
```

Comment:
not speaking cron, how frequent is this?

Reply:
It is every 8 hours, copied from the existing openstack jobs which this would synchronise with. Will talk to the release team about why openstack is so frequent and whether being in sync is going to cause issues

Reply:
https://crontab.guru